osrf_testing_tools_cpp/cmake/osrf_testing_tools_cpp_get_googletest_versions.cmake

```diff
@@ -33,18 +33,22 @@ macro(osrf_testing_tools_cpp_get_googletest_versions output_versions output_loca
     1.7.0
     1.8.1
     1.10.0
+    # Manually patched by osrf
+    1.10.9000
```

Comment:
Sorry to jerk this around, but after thinking about it more, I think I was wrong and your original patch was right. Because if we do this, then when 1.10.1 comes out we won't be able to use it and we'll have to continue with 1.10.9010 or something until 1.11.0 comes out...

Also CMake's version comparison supports tweak (as in major.minor.patch.tweak):

https://cmake.org/cmake/help/latest/variable/CMAKE_VERSION.html

So sorry, but maybe we should revert to your previous change. :(((